prow/tide/tide.go

```diff
@@ -987,6 +1054,7 @@ type PullRequest struct {
 	}
 	HeadRefOID githubql.String `graphql:"headRefOid"`
 	Mergeable  githubql.MergeableState
+
```

Comment:
nit: You've got a couple of these newline diffs, maybe remove?

Reply:
done